library/src/main/java/com/google/maps/android/data/Renderer.java

```diff
@@ -16,8 +16,11 @@
 
 package com.google.maps.android.data;
 
-import android.content.Context;
+import android.app.Activity;
```

Comment:
Could you please use the Android X libraries here instead of the older `android.app` references?

Reply:
Done